src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -62,34 +62,34 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.4.1" />
-    <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.2.0" />
+    <PackageReference Include="Azure.Data.Tables" Version="12.8.1" />
+    <PackageReference Include="Azure.Identity" Version="1.10.0" />
+    <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.5.0" />
+    <PackageReference Include="Azure.Storage.Files.Shares" Version="12.15.0" />
```

Comment:
What is the reasoning behind adding these new packages?

Reply:
Azure.Identity and Azure.Security.KeyVault.Secrets were updated.
Azure.Data.Tables and Azure.Storage.Files.Shares are added to replace deprecated pacakges.